### PR TITLE
Revert "remove payment_type: unscheduled"

### PR DESCRIPTION
**Pull request by harish-venkataramanan-cko**
Reverts checkout/checkout-api-reference#704

https://checkout.slack.com/archives/C0457MNGU8H/p1674641577900379

This was taken out to fix a bug and wait for the merchant comms to be sent. We need to add it again.

### CODE REVIEW

**Comment by sebastian-garcia-cardona-cko** [comment]
Hey @harish-venkataramanan-cko, could you update the `changelog.md` file in `abc_spec` and `nas_spec` please to mention this update please?

**Review by sebastian-garcia-cardona-cko — DISMISSED** [review]

File: nas_spec/changelog.md [hunk 1]
```diff
@@ -2,7 +2,22 @@
 
 | Date       | Description of change                                                                                                 |
 |------------|-----------------------------------------------------------------------------------------------------------------------|
-| 2023/01/25 | Added `Unscheduled` payment_type to payment request                                                                   |
+| 2023/02/22 | Added `3ds.exemption` and `3ds.allow_upgrade` to Hosted Payments Page and Payment Links.
+| 2023/02/16 | Updated PlatformsFileRetrieveResponse and tidied up platforms-files paths                                             |
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/changelog.md:6]:
```suggestion
| 2023/02/16 | Updated `PlatformsFileRetrieveResponse` and tidied up platforms-files paths                                             |
```

File: nas_spec/changelog.md [hunk 2]
```diff
@@ -2,7 +2,22 @@
...
+| 2023/02/16 | Updated `PaymentRequest`, `Address`. Removed unused properties in `PaymentRequestGiropaySource`.                      |
+| 2023/02/15 | Added `exemption_applied` to 3DS details in GET response                                                              |
+| 2023/02/14 | Updated `PaymentRequestProcessingSettings` to add `line_of_business` field to the NAS payment request                 |
+| 2023/02/13 | Update ProcessingData to add `aft`, `merchant_category_code`, `scheme_merchant_id` properties in get PaymentResponse  |
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/changelog.md:10]:
```suggestion
| 2023/02/13 | Update `ProcessingData` to add `aft`, `merchant_category_code`, `scheme_merchant_id` properties in get PaymentResponse  |
```

File: nas_spec/changelog.md [hunk 3]
```diff
@@ -3,11 +3,11 @@
...
 |------------|-----------------------------------------------------------------------------------------------------------------------|
 | 2023/02/22 | Added `3ds.exemption` and `3ds.allow_upgrade` to Hosted Payments Page and Payment Links.
-| 2023/02/16 | Updated PlatformsFileRetrieveResponse and tidied up platforms-files paths                                             |
+| 2023/02/16 | Updated `PlatformsFileRetrieveResponse` and tidied up platforms-files paths                                             |
 | 2023/02/16 | Updated `PaymentRequest`, `Address`. Removed unused properties in `PaymentRequestGiropaySource`.                      |
 | 2023/02/15 | Added `exemption_applied` to 3DS details in GET response                                                              |
 | 2023/02/14 | Updated `PaymentRequestProcessingSettings` to add `line_of_business` field to the NAS payment request                 |
-| 2023/02/13 | Update ProcessingData to add `aft`, `merchant_category_code`, `scheme_merchant_id` properties in get PaymentResponse  |
+| 2023/02/13 | Update `ProcessingData` to add `aft`, `merchant_category_code`, `scheme_merchant_id` properties in get PaymentResponse  |
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/changelog.md:10]:
```suggestion
| 2023/02/13 | Update `ProcessingData` to add `aft`, `merchant_category_code`, `scheme_merchant_id` properties in `GetPaymentResponse`  |
```

Comment (eoin-shaughnessy-cko) [reply nas_spec/changelog.md:10]: Sorry @harish-venkataramanan-cko, think I added this just after you'd added in the other suggestions! Last one, I promise. :)

**Review by eoin-shaughnessy-cko — APPROVED** [review]

**Pull request merged**